backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -437,6 +459,7 @@ export class ExperimentService {
     user: User,
     logger: UpgradeLogger
   ): Promise<ExperimentDTO[]> {
+    this.cacheService.resetCache();
```

Comment:
You should not reset the whole cache instead find the keys with the pattern and reset only those.

Reply:
code updated